src/components/key-value-label/key-value-label.css

```diff
@@ -0,0 +1,16 @@
+.kvLabel {
```

Comment:
Not sure if you just prefer camelcase classes, but you can use dashes here too. In the component you'd just do `styles['kv-label']`;